packages/core/src/blocks/TableBlockContent/TableBlockContent.ts

```diff
@@ -160,10 +159,10 @@ export const Table = createBlockSpecFromStronglyTypedTiptapNode(
     TableExtension,
     TableParagraph,
     TableHeader.extend({
-      content: "tableContent",
+      content: "tableContent*",
```

Comment:
What's the reason for this change? Since table cells should only be able to have a single paragraph in them